spec/defines/snippet_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+describe 'bird::snippet', type: :define do
```

Comment:
Technically the `type: define` is also inferred from the path by rspec-puppet so this should be equivalent
```suggestion
describe 'bird::snippet' do
```